lib/scripts/pdf_a4_portrait.js

```diff
@@ -153,11 +154,19 @@ function createSection (section, content, options) {
     height: options.height,
     contents: phantom.callback(function (pageNum, numPages) {
       var html = o[pageNum] || c[pageNum]
+
+      var pageNumFinal = pageNum, numPagesFinal = numPages;
+      if(opts.paginationStartPage && opts.paginationStartPage > 1) {
+        var pageOffset = opts.paginationStartPage - 1;
+        pageNumFinal = (pageNumFinal - pageOffset > 0 ? pageNumFinal - pageOffset : '');
+        numPagesFinal -= pageOffset;
```

Comment:
I suggest to never use `-=`. More explicit is better.
Would be nice if we can calculate those numbers differently.

e.g.
```
var paginationStartPage = opts.paginationStartPage >  1 ? opts.paginationStartPage : 1
var pageNumFinal = (pageNum - pageOffset > 0) ? pageNumFinal - pageOffset : ''
var numPagesFinal = numPages - (paginationStartPage - 1)
```